DataFormats/Detectors/TRD/src/RawData.cxx

```diff
@@ -98,15 +98,15 @@ std::ostream& operator<<(std::ostream& stream, const TrackletMCMData& tracklet)
 }
 void printTrackletMCMData(o2::trd::TrackletMCMData& tracklet)
 {
-  LOGF(INFO, "TrackletMCMData: Raw:0x%08x pos:%d slope:%d pid:0x%08x checkbit:0x%02x",
-       tracklet.word, tracklet.pos, tracklet.slope, tracklet.pid, tracklet.checkbit);
+//  LOGF(INFO, "TrackletMCMData: Raw:0x%08x pos:%d slope:%d pid:0x%08x checkbit:0x%02x",
```

Comment:
LOGF is not thread safe I take it? I can change this to LOG(info) instead? 
can leave this commented out for now though.

Reply:
No, I actually fixed all of them in a different PR. will rebase this one.